docs/quickstart.rst

```diff
@@ -21,14 +21,14 @@ a good amount of free space on the filesystem that has your backup repository
 (and also on ~/.cache). A few GB should suffice for most hard-drive sized
 repositories. See also :ref:`cache-memory-usage`.
 
-Borg doesn't use space reserved for root on repository disks (even when run as root),
+|project_name| doesn't use space reserved for root on repository disks (even when run as root),
 on file systems which do not support this mechanism (e.g. XFS) we recommend to
-reserve some space in Borg itself just to be safe by adjusting the
+reserve some space in |project_name| itself just to be safe by adjusting the
 ``additional_free_space`` setting in the ``[repository]`` section of a repositories
 ``config`` file. A good starting point is ``2G``.
 
 If |project_name| runs out of disk space, it tries to free as much space as it
-can while aborting the current operation safely, which allows to free more space
+can while aborting the current operation safely, which allows it to free more space
 by deleting/pruning archives. This mechanism is not bullet-proof in some
```

Comment:
the "allows to free" refers to the admin who can choose to run delete/prune then if he/she likes.

borg won't do that by itself / automatically, so no "it", IMHO.

Reply:
"allows to free" is not correct grammar, so it has to be "allows one to free more space", "allows the user to free more space" or "allows more space to be freed".

Reply:
ah, thanks!